source/clear-linux/concepts/bundles-about.rst

```diff
@@ -0,0 +1,31 @@
+.. _bundles-about:
+
+Bundles
+#######
+
+Linux-based operating systems contain the code of several hundred, if
+not thousands, of open source projects. To make this manageable,
+distributions use a concept called "packages" to configure and compile
+the source code of these projects into binaries.
+
+Many distributions then split the content of these compiled packages
+into so-called sub-packages, which are the granularity at which these
+distributions deploy their software. With those kinds of distributions,
+system administrators can then install and update sub-packages
+individually or as a set, using tools such as "yum" and "apt-get."
+
+The Clear Linux\* OS for Intel® Architecture takes a slightly different
+approach. While we also use the concept of packages to manage compiling
+source code into binaries, we do not use the package concept to deploy
+software. Instead, we provide "bundles" that  provide a set of functionality
```

Comment:
The sentence that starts on this line is confusing and should be rewritten

Reply:
@rcaballeromx can you confirm the following suggestion captures what you are saying here:
Instead, we provide "bundles" that provide a set of functionality to the system administrator. The concept of sub-packages does not exist within |CL| as a bundle can contain code from multiple open source projects.

Reply:
IMO, it does not. The ideas the sentence is trying to convey are: 1. We provide bundles not packages. 2. Those bundles provide a complete functionality. 3. Bundles contain as many or as few open source projects as needed to provide that functionality. 
The sentence has nothing to do with sub-packages since those do not exist on CL.

Reply:
Change made